src/bartiq/symbolics/sympy_interpreter.py

```diff
@@ -142,9 +142,15 @@ def eval(cls, p, n):
 class nlz(Function):
     @classmethod
     def eval(cls, n):
+        from sympy import Float, Integer
```

Comment:
issue: do not use imports inside function's body.  Use imports at the top of the file.
Also, in this case we're already importing both `Float` and `Integer` at the top of the file, so this is unnecessary.